disdrodb/tests/test_standards.py

```diff
@@ -0,0 +1,120 @@
+
+def test_get_sensor_variables():
+    function_return = standards.get_sensor_variables()
+    assert function_return ==
+
+def test_get_data_format_dict():
+    function_return = standards.get_data_format_dict()
+    assert function_return ==
+
+def test_get_description_dict():
+    function_return = standards.get_description_dict()
+    assert function_return ==
+
+def test_get_long_name_dict():
+    function_return = standards.get_long_name_dict()
+    assert function_return ==
+
+def test_get_units_dict():
+    function_return = standards.get_units_dict()
+    assert function_return ==
+
+def test_get_sensor_name_dict():
+    function_return = standards.get_sensor_name_dict()
+    assert function_return ==
+
+def test_get_diameter_bins_dict():
+    function_return = standards.get_diameter_bins_dict()
+    assert function_return ==
+
+def test_get_velocity_bins_dict():
+    function_return = standards.get_velocity_bins_dict()
+    assert function_return ==
+
+def test_get_L0A_dtype():
+    function_return = standards.get_L0A_dtype()
+    assert function_return ==
+
+def test_get_L0A_encodings_dict():
+    function_return = standards.get_L0A_encodings_dict()
+    assert function_return ==
+
+def test_get_L0B_encodings_dict():
+    function_return = standards.get_L0B_encodings_dict()
+    assert function_return ==
+
+def test_get_time_encoding():
+    function_return = standards.get_time_encoding()
+    assert function_return ==
+
+def test_set_DISDRODB_L0_attrs():
+    function_return = standards.set_DISDRODB_L0_attrs()
+    assert function_return ==
+
+def test_get_diameter_bin_center():
+    function_return = standards.get_diameter_bin_center()
+    assert function_return ==
+
+def test_get_diameter_bin_lower():
+    function_return = standards.get_diameter_bin_lower()
+    assert function_return ==
+
+def test_get_diameter_bin_upper():
+    function_return = standards.get_diameter_bin_upper()
+    assert function_return ==
+
+def test_get_diameter_bin_width():
+    function_return = standards.get_diameter_bin_width()
+    assert function_return ==
+
+def test_get_velocity_bin_center():
+    function_return = standards.get_velocity_bin_center()
+    assert function_return ==
+
+def test_get_velocity_bin_lower():
+    function_return = standards.get_velocity_bin_lower()
+    assert function_return ==
+
+def test_get_velocity_bin_upper():
+    function_return = standards.get_velocity_bin_upper()
+    assert function_return ==
+
+def test_get_velocity_bin_width():
+    function_return = standards.get_velocity_bin_width()
+    assert function_return ==
+
+def test_get_raw_field_nbins():
+    function_return = standards.get_raw_field_nbins()
+    assert function_return ==
+
+def test_get_raw_field_dim_order():
+    function_return = standards.get_raw_field_dim_order()
+    assert function_return ==
+
+def test_get_raw_spectrum_ndims():
+    function_return = standards.get_raw_spectrum_ndims()
+    assert function_return ==
+"""

```

Comment:
@regislon do we want to test standards? What's the reason for not implementing these drafted functions ?

Reply:
Yes, this is tested in another file. I keep this for consistency. In the other file we don't test function, but directly the content of config file. That why I keep the file commented.